### ISSUE TITLE
Release version 0.1.0

### ISSUE THREAD

**Issue by dlorenc** [kubernetes/minikube#93]
Once we've fixed all the issues in the 0.1 milestone we should do a build and release. We'll have to build and tag an ISO as well as the minikube binaries. Right now it's only been tested on Mac and Linux, so we should stick to those. Windows support can come later.

**Comment by dlorenc** [comment]
Integration tests are passing at https://github.com/kubernetes/minikube/commit/73709243108cc9eba1f034c4c8f99bfe3ca5d15d

If there are no objections I'll cut a release here tomorrow.

**Comment by luxas** [comment]
Remember to bump version in version.go before cutting the release

**Comment by luxas** [comment]
At least #126 should be in the first release, and #119 would be a nice to have.

**Comment by luxas** [comment]
Yes, now only a version bump in the Makefile should be done before cutting

**Comment by dlorenc** [comment]
Released here: https://github.com/kubernetes/minikube/releases/tag/v0.1

**Comment by luxas** [comment]
@dlorenc Why wasn't the version updated?
The released binary on github now outputs the wrong version which could be confusing for users:
```console
$ minikube version
minikube version: v0.0.1
```

You might want to update the binaries...

**Comment by luxas** [comment]
I would also suggest always using `vX.Y.Z` version numbers.

If you feel you have time, you might want to update the version number, re-tag with `v0.1.0` and upload new binaries. That's just a suggestion, feel free to ignore this time :)

**Comment by dlorenc** [comment]
Ah sorry, I meant to call this 0.0.1, to match the github tag/release. Maybe I can just rename the release to match that. I'll play with it tomorrow.

**Comment by luxas** [comment]
Okay, but I think the recommended starting version is `v0.1.0`, and IMO we should use that
e.g. dashboard did this in a good way: https://github.com/kubernetes/dashboard/releases/tag/v0.1.0

We've stated everywhere that the initial dev release will be `0.1`, which equals `v0.1.0` (not `v0.0.1`)

I think you should remove the tag `v0.1`, send a pr that bumps to `v0.1.0`, rebuild the binaries, tag it `v0.1.0`, push and click `New Release` in Github, mark it pre-release and paste back what you've already written in release notes.

**Comment by dlorenc** [comment]
Sure, sent #131

**Comment by dlorenc** [comment]
Released here: https://github.com/kubernetes/minikube/releases/tag/v0.1.0

**Comment by luxas** [comment]
Great! Thanks.
